service/src/main/java/org/databiosphere/workspacedataservice/controller/JobController.java

```diff
@@ -2,6 +2,7 @@
 
 import static org.databiosphere.workspacedataservice.generated.GenericJobServerModel.StatusEnum;
 
+import jakarta.annotation.Nullable;
```

Comment:
Other files use the Spring annotation.

```suggestion
import org.springframework.lang.Nullable
```